worker/include/RTC/Transport.h

```diff
@@ -50,8 +50,7 @@ namespace RTC
 		void AddRtpReceiver(RTC::RtpReceiver* rtpReceiver);
 		void RemoveRtpReceiver(RTC::RtpReceiver* rtpReceiver);
 		void SendRtpPacket(RTC::RtpPacket* packet);
-		void SendRtcpPacket(RTC::RTCP::Packet* packet);
-		void SendRtcpPacket(RTC::RTCP::CompoundPacket* packet);
+		void SendRtcp(const uint8_t* data, size_t len);
```

Comment:
I understand that there should be two methods here, one to send RTCP packets and other to send RTCP compound packets. Yes, they can be joined into a single one (by just passing the pointer and the length) but IMHO it does not look nice. I suggest having both `SendRtcpPacket(RTC::RTCP::Packet)` and `SendRtcpCompoundPacket(RTC::RTCP::CompoundPacket)`.